float64 pos_x
float64 pos_y
string nombre
float64 valor
